include/opt-sched/Scheduler/hash_table.h

```diff
@@ -627,9 +627,20 @@ template <class T> inline void BinHashTable<T>::CmputConsts_() {
 
 template <class T>
 inline UDT_HASHVAL BinHashTable<T>::HashKey(UDT_HASHKEY key) {
-  if (keyBitCnt_ == hashBitCnt_)
-    return (UDT_HASHVAL)key;
-  return ((UDT_HASHVAL)key >> hashRShft_);
+
+// #ifdef IS_DEBUG_SEARCH_ORDER
```

Comment:
Can you remove the commented code -- I realize we already have a lot, but we should be trying to remove it.